lib/Onnxifi/Base.h

```diff
@@ -16,6 +16,7 @@
 #ifndef GLOW_ONNXIFI_BASE_H
 #define GLOW_ONNXIFI_BASE_H
 
+#include "glow/Backends/Backend.h"
 #include "glow/ExecutionEngine/ExecutionEngine.h"
```

Comment:
Do you need to include Backend.h? Can it be a forward declaration?

Reply:
This is needed for `glow::createBackend()`